Makefile

```diff
@@ -36,7 +36,7 @@ build:
 	cp ./docker/config/webapp.env.template ./docker/config/webapp.env
 	sed -i -e 's/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/webapp.env
 
-	${DC} build webapp
+	"${DC}" build webapp
```

Comment:
[nitpick/question/note to myself] I would try to quote this variable once during its initialization.

Reply:
Anecdote: We used to have `\\\\\\\\` or `\\\\\\\\\\\\\\\\` in the Firefox build system to ensure that expanding a variable N times would still have a path separator on Windows.

I think it's good practice to ensure quoting at the call sites, if possible.